packages/components/src/tabs/types.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+import type * as Ariakit from '@ariakit/react';
+
+/**
+ * Internal dependencies
+ */
+import type { IconType } from '../icon';
+
+export type TabsContextProps =
+	| {
+			/**
+			 * The tabStore object returned by Ariakit's `useTabStore` hook.
+			 */
+			store: Ariakit.TabStore;
+			/**
+			 * The class name to add to the active tab.
+			 */
+			activeClass: string;
+			/**
+			 * The unique id string for this instance of the Tabs component.
+			 */
+			instanceId: string;
+	  }
+	| undefined;
+
+export type TabsProps = {
+	/**
+	 * The children elements, which should be at least a
+	 * `Tabs.Tablist` component and a series of `Tabs.TabPanel`
+	 * components.
+	 */
+	children: React.ReactNode;
+	/**
+	 * The class name to add to the active tab.
+	 *
+	 * @default 'is-active'
+	 */
+	activeClass?: string;
```

Comment:
I like that idea: 3b318693a4

I've updated the relevant unit test to also check for that prop instead of the class. Let me know what you think, including if there's a better place in the docs to mention this.

Reply:
We can probably delete that unit test, since I believe that every time we use `getSelectedTab()` we're actually checking the `aria-selected` attribute. Let's double-check, but in case we can delete that test altogether

Reply:
Right you are. Sending that test to ride off into the sunset...